### ISSUE TITLE
Showing date as opposed to seconds for analog widgets

### ISSUE THREAD

**Issue by neunzehnachtneun** [BlackyHawky/Clock#100]
Please add the option to show the date like google clock does with the analog scallop widget.

Thanks!

**Comment by SuvanshRao** [comment]
yes and make it rotate along with the second dot like how it happens in google clock
